src/shared/gpt.c

```diff
@@ -192,7 +192,7 @@ const GptPartitionType gpt_partition_type_table[] = {
 
 static const GptPartitionType *gpt_partition_type_find_by_uuid(sd_id128_t id) {
 
-        FOREACH_ARRAY(t, gpt_partition_type_table, ELEMENTSOF(gpt_partition_type_table) - 1)
```

Comment:
This should not be switched over.

Reply:
Or, drop the last empty element in the table, if that works.

Reply:
Exported? In what sense?

Reply:
`gpt_partition_type_table` is used outside of `shared/gpt.c`. ELEMENTSOF/sizeof can't be used there.

Reply:
Indeed...

Reply:
fixed